### PR TITLE
Added files on ph>edu via upload

### PR DESCRIPTION
**Pull request by Jhn-J**
i uploaded the txt file request on domain/lib > ph > edu

file name: firstasia.txt

### CODE REVIEW

**Comment by philipto** [comment]
Please kindly provide the full name of your school on the first line of the file in your request

**Comment by Jhn-J** [comment]
> Please kindly provide the full name of your school on the first line of the file in your request

sorry, i will reupload the file. there is only a spelling error

**Comment by Jhn-J** [comment]
> Please kindly provide the full name of your school on the first line of the file in your request

I've inserted my school's full name ^ _ ^

**Comment by philipto** [comment]
@Jhn-J First stage of verification passed, official domain is [firstasia.edu.ph](https://firstasia.edu.ph) as far as I verified. This is just an informational note. Pull request is still in the review. I appreciate your patience while waiting for the final decision.

**Comment by Jhn-J** [comment]
yes, that's the official domain.

this is noted. thanks!

File: lib/domains/ph/edu/firstasia.txt
Comment (Jhn-J) [inline lib/domains/ph/edu/firstasia.txt:1]: First Asia Institute of Technology and Humanities

School's official domain: https://www.firstasia.edu.ph/

**Comment by LubimAlex** [comment]
Hello @Jhn-J, Pull request merged. Thank you!

**Pull request merged**